management/backup.py

```diff
@@ -417,21 +417,31 @@ def list_target_files(config):
 					ret.append( (match.groups()[1], int(match.groups()[0].replace(',',''))) )
 			return ret
 		else:
-			raise ValueError("Connection to rsync host failed")
-
-	elif p.scheme == "s3":
+			if 'Permission denied (publickey).' in listing:
+				reason = "Invalid user or check you correctly copied the SSH key."
+			elif 'Network is unreachable' in listing:
+				reason = "The IP address provided is unreachable."
```

Comment:
The user input may not have been an ip address. Better to display the actual output to the user than to try to make it more friendly because the friendly part may be wrong.